terraform/python/ec2/main.tf

```diff
@@ -46,7 +46,7 @@ data "aws_ami" "ami" {
   most_recent      = true
   filter {
    name = "name"
-   values = ["al2023-ami-2023.3.20240117.0-kernel-6.1-x86_64"]
```

Comment:
@zzhlogin is there a reason why we decided to use `-kernel-` for the ami? Is it needed for python?

Reply:
Actually should be fine since the test is working